spyder/utils/conda.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Conda/anaconda utilities."""
+
+# Standard library imports
+import os
+import sys
+
+
+def add_quotes(path):
+    """Return quotes if needed for spaces on path."""
+    quotes = '"' if ' ' in path and '"' not in path else ''
+    return '{quotes}{path}{quotes}'.format(quotes=quotes, path=path)
+
+
+def is_conda():

```

Comment:
A couple of comments about this function:

1. Please name it `is_conda_env`.
2. This function needs to receive the path of the Python interpreter in the env (e.g. `/home/carlos/miniconda3/envs/py37/bin/python`), select the directory two levels above it (i.e. `/home/carlos/miniconda3/envs/py37`), which corresponds to `sys.prefix` in that env, and use that directory instead of `sys.prefix` below. That's because right now  you're detecting if Spyder is installed in a conda env, not that the interpreter we're using to start kernels is.

Reply:
Fixed!